checks/raw/binary_artifact_test.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2021 Security Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package raw
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+
+	"github.com/ossf/scorecard/v4/checker"
+	"github.com/ossf/scorecard/v4/clients/githubrepo"
+	"github.com/ossf/scorecard/v4/clients/localdir"
+	"github.com/ossf/scorecard/v4/log"
+)
+
+// TestBinaryArtifact tests the BinaryArtifact checker.
+func TestBinaryArtifacts(t *testing.T) {
+	t.Parallel()
+	tests := []struct {
+		name        string
+		inputFolder string
+		err         error
+		expected    checker.CheckResult
```

Comment:
raw part of check does not return`checker.CheckResult` but the results defined in https://github.com/ossf/scorecard/blob/main/checker/raw_result.go#L35. I think we should be testing for this instead, right?